scripts/generate_gff3_db.py

```diff
@@ -0,0 +1,127 @@
+"""
```

Comment:
Not parsers per GFF3, but converters.  Common syntax, varying
data semantics.

GFF3 is a lot better than GFF2/GTF, because it added definition
of some data sematics, not just syntax.  There is still a wide
range of data that is encoded

There are also other formats we would want to convert to
features directly, such as BED format, and RepeatMasker, and not
convert to GFF3 just for input.

Jerome Kelleher <notifications@github.com> writes:
> In scripts/generate_gff3_db.py:
> 
> > @@ -0,0 +1,127 @@
> > +"""
> 
> Ugh. Are GFF files that awful? It seems a cop out to expect the repo manager
> write their own parser for each GFF file. Anyway, ignore the above comment for
> now, we should discuss this after the initial drop into the sequence
> annotations branch.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub*
>